resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -222,6 +222,13 @@ private[spark] class ExecutorPodsAllocator(
         kubernetesClient.pods().create(podWithAttachedContainer)
         newlyCreatedExecutors(newExecutorId) = clock.getTimeMillis()
         logDebug(s"Requested executor with id $newExecutorId from Kubernetes.")
+        try {
```

Comment:
Bill has some good recommendations here for patching code. https://github.com/lyft/lyft-hadoop#patching.

Do we want to push it to a separate function at least and maybe behind a flag too.

Reply:
Sure will add a new method